services/s3/src/main/java/software/amazon/awssdk/services/s3/endpoints/internal/EndpointAttributeProvider.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.s3.endpoints.internal;
```

Comment:
These s3.endpoints.internal files look new, but they are just copies of the .java.resource files that have already been code generated. I just copied them into S3 because this is the only service that uses them.

You can skip reviewing these files if you want, because they were already reviewed. I didn't change them.